src/NUnitFramework/tests/Constraints/NoItemConstraintTests.cs

```diff
@@ -0,0 +1,60 @@
+﻿// ***********************************************************************
+// Copyright (c) 2007 Charlie Poole, Rob Prouse
+//
```

Comment:
Since this is a new file the year should be 2017, but I don't think that this should hold back the PR.